models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+        description: Operation indicator for wage types 
+      - name: opk04
+        description: Operation indicator for wage types 
+      - name: opk05
+        description: Operation indicator for wage types 
+      - name: opk06
+        description: Operation indicator for wage types 
+      - name: opk07
+        description: Operation indicator for wage types 
+      - name: opk08
+        description: Operation indicator for wage types 
+      - name: opk09
+        description: Operation indicator for wage types 
+      - name: opk10
+        description: Operation indicator for wage types 
+      - name: opk11
+        description: Operation indicator for wage types 
+      - name: opk12
+        description: Operation indicator for wage types 
+      - name: opk13
+        description: Operation indicator for wage types 
+      - name: opk14
+        description: Operation indicator for wage types 
+      - name: opk15
+        description: Operation indicator for wage types 
+      - name: opk16
+        description: Operation indicator for wage types 
+      - name: opk17
+        description: Operation indicator for wage types 
+      - name: opk18
+        description: Operation indicator for wage types 
+      - name: opk19
+        description: Operation indicator for wage types 
+      - name: opk20
+        description: Operation indicator for wage types 
+      - name: opk21
+        description: Operation indicator for wage types 
+      - name: opk22
+        description: Operation indicator for wage types 
+      - name: opk23
+        description: Operation indicator for wage types 
+      - name: opk24
+        description: Operation indicator for wage types 
+      - name: opk25
+        description: Operation indicator for wage types 
+      - name: opk26
+        description: Operation indicator for wage types 
+      - name: opk27
+        description: Operation indicator for wage types 
+      - name: opk28
+        description: Operation indicator for wage types 
+      - name: opk29
+        description: Operation indicator for wage types 
+      - name: opk30
+        description: Operation indicator for wage types 
+      - name: opk31
+        description: Operation indicator for wage types 
+      - name: opk32
+        description: Operation indicator for wage types 
+      - name: opk33
+        description: Operation indicator for wage types 
+      - name: opk34
+        description: Operation indicator for wage types 
+      - name: opk35
+        description: Operation indicator for wage types 
+      - name: opk36
+        description: Operation indicator for wage types
+      - name: opk37
+        description: Operation indicator for wage types
+      - name: opk38
+        description: Operation indicator for wage types 
+      - name: opk39
+        description: Operation indicator for wage types 
+      - name: opk40
+        description: Operation indicator for wage types 
+      - name: ordex
```

Comment:
Are these all intended to be the same description?

Reply:
That appears to be case. 
<img width="728" alt="Screenshot 2023-07-06 at 4 20 48 AM" src="https://github.com/fivetran/dbt_sap_source/assets/108772760/bf547299-3461-4ed1-99b8-ff208a14007e">